### DISCUSSION TITLE
[HW2] Problem 1.3

### DISCUSSION THREAD

**Discussion by emjay73** [seominjoon/kaist-ai605#97] [Q&A] [answered]
Problem 1.3 (2 points) Prove that cosine distance (1 - cosine similarity) is a metric.

Is cosine distance a metric?
Wikipedia says it is not.

> https://en.wikipedia.org/wiki/Cosine_similarity    
> The term cosine distance is often used for the complement in positive space, 
> ...
> It is important to note, however, that this is not a proper distance metric as it does not have the triangle inequality property
> or, more formally, the Schwarz inequality—and it violates the coincidence axiom

Do we have to prove that it is NOT a metric?
or do we have to show that 'angular distance' is a metric?

**Comment by seominjoon** [comment, marked as answer]
I am sorry about the delayed response. My notification seems to be not working after I changed my phone..

This is my mistake. Please prove that it is NOT a metric. Proving that the angular distance is a metric is also acceptable, since this must have confused a lot of you. Sorry about that.
